modules/spark/src/main/scala/notebook/spark/util/SparkClosureUtil.scala

```diff
@@ -0,0 +1,59 @@
+package notebook.spark.util
+
+import scala.reflect.runtime.universe
+import org.apache.log4j.AppenderSkeleton
+import org.apache.log4j.spi.LoggingEvent
+
+class SparkClosureUtil(appendTo: String => Unit = println) {
+
+  // get the closure cleaner
+  private val runtimeMirror = universe.runtimeMirror(getClass.getClassLoader)
+  private val module = runtimeMirror.staticModule("org.apache.spark.util.ClosureCleaner")
```

Comment:
it's `private[spark] object ClosureCleaner`, so why not just move this `SparkClosureUtil` class to a fake spark package (e.g. `org.apache.spark.spark_notebook_hacks`), we'll have cleaner code and even will get type-safety!?

 IMHO this reflection is evil!